docs/add_component_outline.txt

```diff
@@ -40,19 +41,19 @@ class ComponentName(Component):
             s.last = startjunc.copyjunc()
 
         ### create polylines and append to drawing
-        s.drawing.append(sdxf.PolyLine(pts))
-	pts = [(x1,y1),(x2,y2),(x3,y3),(x4,y4)]
-	...
-        
-	### or use existing CPW elements to create component
-	CPWStraight(s,settings = {'length':300,
-				  'pinw':20,
-				  'gapw':8.372
-				  })	
+		pts = [(x1,y1),(x2,y2),(x3,y3),(x4,y4)]
```

Comment:
what's going on with the white space here? Looks like you added a tab to everything but I can't figure out why

Reply:
I probably got confused, I'll fix that.